Src/BootCamp.Chapter/ShuffleDemo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal static class ShuffleDemo
```

Comment:
This is supposed to be named IEnumerable extensions and in fact moved to the same class as other extension.

Reply:
done